src/Uplink/Admin/Auth.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+
+namespace StellarWP\Uplink\Admin;
+
+use StellarWP\Uplink\Config;
+use StellarWP\Uplink\Resources\Collection;
+
+class Auth {
+
+	public function do_auth_html() {
+		$collection = Config::get_container()->get( Collection::class );
+		$plugin     = $collection->current();
+		$license    = $plugin->get_license_object();
+
+		$token   = get_option( sprintf( 'stellarwp_origin_%s_auth_token', $license->origin->slug ?? '' ), '' );
+		$message = esc_html__( 'Connect to origin', '%TEXTDOMAIN%' );
+		$classes = [ 'button', 'button-primary' ];
```

Comment:
Defining the structure here looks too prescriptive for the plugins implementing this. Could we provide a new filter to allow them to override the HTML?